releng/server/standalone/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+chown -R dirigible:dirigible $CATALINA_HOME
+
+exec gosu dirigible $@
```

Comment:
In this commit https://github.com/SAP/xsk/pull/1634/commits/bc5c9c71852e3299dec3a7492c50e3b38702db4a entrypoint.sh is deleted, because in distroless image there is non root user and we don't need to create new one.

Reply:
> In this commit [bc5c9c7](https://github.com/SAP/xsk/commit/bc5c9c71852e3299dec3a7492c50e3b38702db4a) entrypoint.sh is deleted, because in distroless image there is non root user and we don't need to create new one.

yes, dockerfile-related stuff will be updated to follow that changes.